tvx/exec_lotus.go

```diff
@@ -29,38 +30,66 @@ var execLotusCmd = &cli.Command{
 		&cli.StringFlag{
 			Name:        "file",
 			Usage:       "input file",
-			Required:    true,
 			Destination: &execLotusFlags.file,
 		},
+		&cli.BoolFlag{
+			Name:        "stdin",
+			Usage:       "",
+			Destination: &execLotusFlags.stdin,
+		},
 	},
 	Action: runExecLotus,
 }
 
 func runExecLotus(_ *cli.Context) error {
-	if execLotusFlags.file == "" {
-		return fmt.Errorf("test vector file cannot be empty")
-	}
+	switch {
+	case execLotusFlags.stdin == true:
```

Comment:
Is the explicit true comparison needed here? Looks weird.

Reply:
I opted out to have stdin as default case, and if --file is specified, we use the file.